data-otservbr-global/npc/tarun.lua

```diff
@@ -51,6 +51,71 @@ npcType.onCloseChannel = function(npc, creature)
 	npcHandler:onCloseChannel(npc, creature)
 end
 
+local function creatureSayCallback(npc, creature, type, message)
+	local player = Player(creature)
+	local playerId = player:getId()
+
+	if not npcHandler:checkInteraction(npc, creature) then
+		return false
+	end
+
+	if MsgContains(message, "mission") then
+		if player:getStorageValue(Storage.AdventurersGuild.TheLostBrother) < 1 then
+			npcHandler:setTopic(playerId, 1)
+			npcHandler:say(
+				{
+					"My brother is missing. I fear, he went to this evil palace north of here. A place of great beauty, certainly filled with riches and luxury. But in truth it is a threshold to hell and demonesses are after his blood. ...",
+					"He is my brother, and I am deeply ashamed to admit but I don't dare to go there. Perhaps your heart is more courageous than mine. Would you go to see this place and search for my brother?"
+				},
+				npc,
+				creature
+			)
+		elseif player:getStorageValue(Storage.AdventurersGuild.TheLostBrother) == 1 then
+			npcHandler:say("I hope you will find my brother.", npc, creature)
+		elseif player:getStorageValue(Storage.AdventurersGuild.TheLostBrother) == 2 then
```

Comment:
how we have a lot of `player:getStorageValue(Storage.AdventurersGuild.TheLostBrother)`
I think putting this on a variable inside this is better.
`local theLostBrother = player:getStorageValue(Storage.AdventurersGuild.TheLostBrother)`

```suggestion
		local theLostBrother = player:getStorageValue(Storage.AdventurersGuild.TheLostBrother)
		if theLostBrother < 1 then
			npcHandler:setTopic(playerId, 1)
			npcHandler:say(
				{
					"My brother is missing. I fear, he went to this evil palace north of here. A place of great beauty, certainly filled with riches and luxury. But in truth it is a threshold to hell and demonesses are after his blood. ...",
					"He is my brother, and I am deeply ashamed to admit but I don't dare to go there. Perhaps your heart is more courageous than mine. Would you go to see this place and search for my brother?"
				},
				npc,
				creature
			)
		elseif theLostBrother == 1 then
			npcHandler:say("I hope you will find my brother.", npc, creature)
		elseif theLostBrother == 2 then
```

Reply:
done